service/task.go

```diff
@@ -297,6 +298,9 @@ func (uis *UIServer) taskPage(w http.ResponseWriter, r *http.Request) {
 		for _, tr := range projCtx.Context.Task.LocalTestResults {
 			uiTask.TestResults = append(uiTask.TestResults, uiTestResult{TestResult: tr})
 		}
+		if uiTask.PartOfDisplay {
+			uiTask.DisplayTaskID = projCtx.Task.DisplayTask.Id
```

Comment:
nil checks?

Reply:
PartOfDisplay being true guarantees that this is not nil, but I can add them if you'd like